controllers/keyword.go

```diff
@@ -0,0 +1,54 @@
+package controllers
+
+import (
+	"encoding/csv"
+	"net/http"
+	"os"
+	"path/filepath"
+
+	"github.com/gin-gonic/gin"
+	html "github.com/gutakk/go-google-scraper/helpers/html"
+	"github.com/gutakk/go-google-scraper/models"
+)
+
+const (
+	keywordTitle = "Keyword"
+	keywordView  = "keyword"
+)
+
+type KeywordController struct{}
+
+func (k *KeywordController) applyRoutes(engine *gin.Engine) {
+	engine.GET("/keyword", k.displayKeyword)
+	engine.POST("/keyword", k.uploadKeyword)
+}
+
+func (k *KeywordController) displayKeyword(c *gin.Context) {
+	html.RenderWithFlash(c, http.StatusOK, keywordView, keywordTitle, nil)
+}
+
+func (k *KeywordController) uploadKeyword(c *gin.Context) {
+	file, _ := c.FormFile("file")
```

Comment:
How about using a struct like in the registration and login? Having struct to validate params is great even if there is only one param 💪

Reply:
Fixed in 3dcfa7705031d36383aba55681a40f9a7c94ddc0